src/main/java/duke/logic/UiManager.java

```diff
@@ -0,0 +1,199 @@
+package duke.logic;
+
+import java.util.Scanner;
+
+import duke.CommonString;
+import duke.task.DukeTask;
+
+/**
+ * Represents the User interface.
+ * Manages the User Interaction Interface.
+ * It contains <code>Enums</code> for pre-defined texts to be printed for the program
+ * and methods to print feedback to the users for different commands.
+ */
+public class UiManager {
```

Comment:
I like this class because its function has been very well defined and followed through with! Good job! 👍